cmd/server-rlimit.go

```diff
@@ -65,7 +69,7 @@ func GetMaxCacheSize() (cacheSize uint64, err error) {
 
 	// Get total RAM.
 	var stats sys.Stats
-	if stats, err = sys.GetStats(); err != nil {
+	if stats, err = sys.GetStats(os.Getpid()); err != nil {
```

Comment:
As `sys.GetStats()` returns current PID's stat, there is no need to pass PID as argument.